src/clj/rems/workflow/dynamic.clj

```diff
@@ -196,8 +198,150 @@
                              :event/actor "foo"
                              :application/id 123}))))))
 
+;;; Roles and permissions
 
-;;; Events
+(defn hide-sensitive-dynamic-events [events]
```

Comment:
This could be written

```
(defn hide-sensitive-dynamic-events [events]
  (remove (comp #{:application.event/comment-requested
                  :application.event/commented
                  :application.event/decided
                  :application.event/decision-requested}
                :event/type)
          events))
```